controllers/cloud.redhat.com/run.go

```diff
@@ -34,7 +36,12 @@ func init() {
 	//+kubebuilder:scaffold:scheme
 }
 
+//go:embed version.txt
+var Version string
+
 func Run(metricsAddr string, probeAddr string, enableLeaderElection bool) {
+	enoVersion.With(prometheus.Labels{"version": Version}).Inc()
```

Comment:
I don't understand this - and that's likely on me - but it looks like you're just incrementing the version that you're reporting to prom every time the operator starts? If I'm reading that right (and I'm likely not) that wouldn't be what you want, right?

Reply:
I guess you're recording the version as the `version` label? That's probably OK - I bet that's what we do in Clowder.

Reply:
I commented on the next comment thread about this as well.